File: snakes/rufio/tactics/Continue.js
const utils = require('../../../utils');

const Continue = () => {
	return ({ context, state }) => {
//		if (context.turn === 0) utils.LogMove(context.turn, state.move, 'Initial Move');
//		else utils.LogMove(context.turn, state.move, 'Continue');

		return state.move;
	};
};

module.exports = { Continue };

File: snakes/rufio/tactics/Continue.go
package tactics

import (
	snek "github.com/JMTyler/battlesnake/_"
	//"utils"
)

type Continue struct{ Name string }

func (t *Continue) Description() string {
	return t.Name
}

func (tactic *Continue) Run(context snek.Context, state *snek.State) string {
	//if (context.turn === 0) utils.LogMove(context.turn, state.move, 'Initial Move')
	//else utils.LogMove(context.turn, state.move, 'Continue')

	return state.Move
}
